ui/widgets/src/main/java/com/appcoins/wallet/ui/widgets/BackupCardComposable.kt

```diff
@@ -16,59 +18,123 @@ import com.appcoins.wallet.ui.widgets.component.ButtonWithText
 import java.util.Date
 
 @Composable
-fun BackupAlertCard(onClickButton: () -> Unit, hasBackup: Boolean, backupDate: Long = 0L) {
+fun BackupAlertCard(
+  modifier: Modifier = Modifier,
+  onClickButton: () -> Unit,
+  hasBackup: Boolean,
+  backupDate: Long = 0L,
+) {
   AlertCard(
-    onClickButton = onClickButton,
+    modifier = modifier,
+    onClickPositiveButton = onClickButton,
     title = if (hasBackup) R.string.backup_confirmation_no_share_title else R.string.my_wallets_action_backup_wallet,
     message = if (hasBackup)
       stringResource(
         id = R.string.mywallet_backed_up_date,
         DateFormat.format("dd/MM/yyyy", Date(backupDate)).toString()
       )
     else stringResource(R.string.backup_wallet_tooltip),
-    buttonLabel = if (hasBackup) R.string.mywallet_backup_again_button else R.string.action_backup_wallet,
+    positiveButtonLabel = stringResource(id = if (hasBackup) R.string.mywallet_backup_again_button else R.string.action_backup_wallet),
     icon = if (hasBackup) R.drawable.ic_check_circle else R.drawable.ic_alert_circle
   )
 }
 
 @Composable
-fun VerifyWalletAlertCard(onClickButton: () -> Unit, verified: Boolean) {
+fun VerifyWalletAlertCard(
+  onClickButton: () -> Unit,
+  verified: Boolean,
+  waitingCode: Boolean,
+  onCancelClickButton: () -> Unit,
+  modifier: Modifier = Modifier
+) {
   AlertCard(
-    onClickButton = onClickButton,
-    title = if (verified) R.string.verification_settings_verified_title else R.string.referral_verification_title,
-    message = stringResource(R.string.mywallet_unverified_body),
-    buttonLabel = R.string.referral_verification_title,
-    icon = if (verified) R.drawable.ic_check_circle else R.drawable.ic_alert_circle
+    onClickPositiveButton = onClickButton,
+    title = if (verified) R.string.verification_settings_verified_title
```

Comment:
Since this is a composer, wouldn't it be better to take these ifs out to the constructor? and leave this to the responsibility of whoever calls it?